src/renderers/dom/shared/SVGDOMPropertyConfig.js

```diff
@@ -271,6 +271,8 @@ var ATTRS = {
   yChannelSelector: 'yChannelSelector',
   z: 0,
   zoomAndPan: 'zoomAndPan',
+  xmlns: 0,
+  xmlnsXlink: 'xml:xlink',
```

Comment:
2 things:
1. This value should be `'xmlns:xlink'` right?
2. Can you move these up so they're with the other `xml*` attributes? Doesn't have to be perfect, we have a few others minorly out of order so we'll do a `:sort` sometime later.